feature/aft/aft_summary/otg_tests/route_summary_counters_test/route_summary_counters_test.go

```diff
@@ -173,13 +177,17 @@ func TestRouteSummaryWithBGP(t *testing.T) {
 		desc: "propagate IPv6 over IPv6",
 		dut: dutData{
 			routerID: dutPort1.IPv4,
-			neighborPG: map[string]string{
-				otgPort1.IPv6: bgpPeerGroup,
-				otgPort2.IPv6: bgpPeerGroup,
-			},
-			neighborAS: map[string]uint32{
-				otgPort1.IPv6: ateAS1,
-				otgPort2.IPv6: ateAS2,
+			neighborConfig: []*cfgplugins.NeighborConfig{
+				{
+					IPv4Neighbor: otgPort1.IPv6,
```

Comment:
Should the key be `IPv6Neighbor` instead of `IPv4Neighbor` here as `ipv4` flag is `false` ? Considering in `bgp.go` it looks for `IPv6Neighbor` for ipv6 - 

https://github.com/openconfig/featureprofiles/blob/4e3a6a324c00ea626fc565500e696f39901eab6c/internal/cfgplugins/bgp.go#L420-L421

Reply:
you're right, thanks for pointing